dockernet/config.sh

```diff
@@ -18,7 +18,7 @@ KEYS_LOGS=$SCRIPT_DIR/logs/keys.log
 # List of hosts enabled 
 #  `start-docker` defaults to just GAIA if HOST_CHAINS is empty
 # `start-docker-all` always runs all hosts
-HOST_CHAINS=(GAIA) 
+HOST_CHAINS=() 
```

Comment:
shouldn't the default be to start with at least one host?

Reply:
Yeah if it’s empty, it defaults to just GAIA (see comment above). The idea is you should only use that array if you want to override those two defaults.

Lmk if you can think of a different way to set it up that would be more clear!